### PR TITLE
Cleanup badges references

### PR DESCRIPTION
**Pull request by filipgorczynski**
Removes almost all (except CSS related) references to UserBadges.

### CODE REVIEW

**Comment by magul** [comment]
@filipgorczynski You have an pylint error on travis to be fixed.

**Comment by Yog** [comment]
gonna miss them

File: apps/volontulo/tests/common.py [hunk 1]
```diff
@@ -73,22 +71,6 @@ def initialize_filled_volunteer_and_organization():
         'volunteer2'
     )
     volunteer_user2.save()
-    user_profile2 = UserProfile.objects.create(user=volunteer_user2)
```
Comment (magul) [inline apps/volontulo/tests/common.py:76]: @filipgorczynski that's the reason, why travis is failing. Are You able to readd this particular line (without assigning to `user_profile2`)?

**Comment by magul** [comment]
I'm taking it over and will create new PR from my fork for it.

**Pull request closed without merging**